community-plugins.json

```diff
@@ -1013,5 +1013,12 @@
         "description": "Open or reveal the current note in DEVONthink.",
         "author": "ryanjamurphy",
         "repo": "ryanjamurphy/DEVONlink-obsidian"
+    },
+    {
+        "id": "imdone-obsidian-plugin",
+        "name": "Open cards in imdone from obsidian.",
```

Comment:
Didn't want to just name it "imdone"? 🤔

Reply:
Good point, but I didn't want confusion with the product and already have a repo named `imdone/imdone`.